### PR TITLE
Indicate Safari 13 unprefixed support for sticky

### PR DESCRIPTION
**Pull request by chrishtr**
[Data here](https://wpt.fyi/results/css/css-position?diff&filter=ADC&q=sticky&run_id=311390003&run_id=308980004).

### CODE REVIEW

File: css/properties/position.json [hunk 1]
```diff
@@ -257,14 +257,24 @@
...
-              },
+              "safari": [
+                {
+                  "prefix": "-webkit-",
+                  "version_added": "6.1"
+                },
+                {
+                  "version_added": "13"
+                }
+              ],
```
Comment (sideshowbarker) [inline css/properties/position.json:268]:
```suggestion
              "safari": [
                {
                  "version_added": "13"
                },
                {
                  "prefix": "-webkit-",
                  "version_added": "6.1"
                }
              ],
```
See https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#the-support_statement-object:
> sort such arrays with entries applying to the most recent browser releases first and sort entries with prefixes or flags after those without

File: css/properties/position.json [hunk 2]
```diff
@@ -257,14 +257,24 @@
...
+              "safari_ios": [
+                {
+                  "prefix": "-webkit-",
+                  "version_added": "6.1"
+                },
+                {
+                  "version_added": "13"
+                }
+              ],
```
Comment (sideshowbarker) [inline css/properties/position.json:277]:
```suggestion
              "safari_ios": [
                {
                  "version_added": "13"
                },
                {
                  "prefix": "-webkit-",
                  "version_added": "6.1"
                }
              ],
```

**Review by sideshowbarker — CHANGES_REQUESTED** [review]

**Comment by chrishtr** [comment]
Applied suggested changes.

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
Thanks much and welcome to the browser-compat-data project 🎉

**Pull request merged**